subiquity/models/network.py

```diff
@@ -34,53 +34,86 @@ def render_config(self):
         else:
             return super().render_config()
 
+    def _host_combined_cloud_config(self) -> dict:
```

Comment:
I see what you mean about a utility function.  Yes, we'll want to share this with the server/server.py code.  Maybe lift this to a new file subiquity/cloudinit.py ?  Or I'm happy to extract this for my autoinstall consume-combined-cloud-config branch if we don't do it here.

Reply:
Moved this function to `subiquity.cloudinit` if needed in the future we can define a class for interacting w/ cloud-init on the host